### PR TITLE
Feature/54050 popup version

### PR DESCRIPTION
**Pull request by kleyla**
## Description

Popup for version and functionalities added.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|   | :hammer: Refactoring       |
|  ✓ | :100: Add tests            |
|   | :link: Update dependencies |
|   | :scroll: Docs              |

### CODE REVIEW

File: src/config/AxiosInstance.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const AxiosInstance = axios.create({
+  baseURL: 'http://localhost:3000',
```
Comment (Carlos-YD) [inline src/config/AxiosInstance.ts:4]: @kleyla This information should not be hardcoded, at least can be moved to variables or better to environment variables.

**Review by Carlos-YD — CHANGES_REQUESTED** [review]

File: src/config/AxiosInstance.ts (hunk 1 above)
Comment (kleyla) [reply src/config/AxiosInstance.ts]: @Carlos-YD  I have fixed that and deleted the library that appears in alert from security of sonarcloud, thank you for the review :)

**Review by Carlos-YD — CHANGES_REQUESTED** [review]
We still have pending:
- 1 security hotspot
- Missing unit tests to pass the code coverage check
- No other team member reviewed the code

**Comment by kleyla** [comment]
> We still have pending:
> 
>     * 1 security hotspot
> 
>     * Missing unit tests to pass the code coverage check
> 
>     * No other team member reviewed the code

Thank you.. I have fixed "1 security hotspot", about code coverage, in this task I have not added any Model so I am not sure if should add any test

**Review by FavioSauto — APPROVED** [review]

**Review by amr-jorge — APPROVED** [review]
todo bien

**Review by Carlos-YD — APPROVED** [review]

**Pull request merged**